Where_Is_My_Wife/Assets/_Utilities/Templates/SingletonDestroyOnLoad.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+public class SingletonDestroyOnLoad<T> : MonoBehaviour where T : MonoBehaviour
```

Comment:
Dont do this, dont respects the DRY pattern, just add a `[SerializeField] bool` field to know if it should be destroyed on load or not.